src/_container.scss

```diff
@@ -0,0 +1,10 @@
+
+@mixin vui-gs-container() {
```

Comment:
I am working on the breakpoints currently.  This is subject to change as a result.